.bazelrc

```diff
@@ -1,16 +1,18 @@
 startup --host_jvm_args=-Xmx8g
 startup --unlimit_coredumps
 
+build --color=yes
```

Comment:
build or build:ci? Will `build` applies to `build:ci`, too? Do we always keep color on for all builds, or only enable it on ci?

Reply:
I enable ```color``` when to be on CI.